components/tools/OmeroPy/test/integration/gatewaytest/test_get_objects.py

```diff
@@ -597,8 +614,10 @@ def testTraversal(self, author_testimg_tiny):
         assert len(pr.listParents()) == 0
 
     def testListOrphans(self, gatewaywrapper):
-        gatewaywrapper.loginAsUser()
-        eid = gatewaywrapper.gateway.getUserId()
+        # We login as Author, since they have other non-orphan images
+        gatewaywrapper.loginAsAuthor()
```

Comment:
Should we deprecate that method and introduce ``loginAsOwner`` (if author means owner), is that a legacy for carlos's work?